graylog2-shared/src/main/java/org/graylog2/shared/rest/CORSFilter.java

```diff
@@ -35,4 +40,23 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
             responseContext.getHeaders().add("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE, OPTIONS");
```

Comment:
Does it make sense to include the `Access-Control-Max-Age`-header in each response so that caching time for suitable requests is enlengthened?